src/util/schema_utils.js

```diff
@@ -4,7 +4,7 @@
 const _ = require('lodash');
 const util = require('util');
 
-const COMMON_SCHEMA_KEYWORDS = ['doc', '$id'];
+const COMMON_SCHEMA_KEYWORDS = ['doc', '$id', 'maxItems'];
```

Comment:
doesn't seem like this is in the right level, it should be in the array type case only

Reply:
Since I will not add this property to the schema `maxItems`, this change will be canceled anyway.